Files/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -24,6 +26,11 @@ public DrivesWidget()
             InitializeComponent();
         }
 
+        private async void EjectDevice_Click(object sender, RoutedEventArgs e)
+        {
+            await Interaction.EjectDeviceAsync(((MenuFlyoutItem)sender).Tag.ToString());
```

Comment:
Ideally we should derive from DriveItem here.

Reply:
Hi @yaichenbaum I don't find anything particularly wrong with the current approach.
Anyway an alternative approach is in #2347, if you like it more